lib/straight/order.rb

```diff
@@ -143,6 +158,13 @@ def status_changed?
         @status_changed
       end
 
+      def tid
+        (respond_to?(:[]) ? self[:tid] : @tid) || begin
+          tids = (accepted_transactions || []).map { |t| t[:tid] }.join(',')
+          tids.empty? ? nil : tids
+        end
+      end
```

Comment:
It's a temporal solutions for backward-compatibility. Field `tid` is always `nil` from now on.